src/host/CommandListPopup.cpp

```diff
@@ -45,9 +45,9 @@ static COORD calculatePopupSize(const CommandHistory& history)
     }
 
     // calculate height, it can range up to 20 rows
-    auto height = std::clamp(gsl::narrow<short>(history.GetNumberOfCommands()), minSize.Y, 20i16);
+    auto height = std::clamp(gsl::narrow<til::CoordType>(history.GetNumberOfCommands()), minSize.Y, 20);
 
-    return { gsl::narrow<short>(width), height };
+    return { gsl::narrow_cast<til::CoordType>(width), height };

```

Comment:
you kept one of these as a throwing narrow and changed the other into a non-throwing narrow?

Reply:
`width` can't be out of range, because it's initialized as 40, can only grow as it's only updated with `std::max` and before this line it says:
```cpp
    if (width > SHRT_MAX)
    {
        width = SHRT_MAX;
    }
```